src/routes/package-create-route/package-create.test.js

```diff
@@ -0,0 +1,152 @@
+import { MemoryRouter } from 'react-router';
+
+import {
+  render,
+  cleanup,
+  act,
+  fireEvent,
+} from '@testing-library/react';
+
+import PackageCreateRoute from './package-create';
+import Harness from '../../../test/jest/helpers/harness';
+
+const mockHistory = {
+  replace: jest.fn(),
+  goBack: jest.fn(),
+};
+
+const location = {
+  pathname: 'pathname',
+  search: '',
+  hash: '',
+};
+
+const createRequest = {
+  timestamp: 0,
+  type: 'create',
+  params: {},
+  isPending: false,
+  isResolved: false,
+  isRejected: false,
+  records: [],
+  meta: {},
+  errors: [],
+};
+
+const accessStatusTypes = {
+  isDeleted: false,
+  isLoading: false,
+  items: {
+    data: {},
+  },
+};
+
+const mockGetAccessTypes = jest.fn();
+const mockCreatePackage = jest.fn();
+const mockRemoveCreateRequests = jest.fn();
+
+const getPackageCreateRoute = (props = {}) => (
+  <MemoryRouter>
+    <Harness>
+      <PackageCreateRoute
+        accessStatusTypes={accessStatusTypes}
+        createPackage={mockCreatePackage}
+        createRequest={createRequest}
+        getAccessTypes={mockGetAccessTypes}
+        history={mockHistory}
+        location={location}
+        removeCreateRequests={mockRemoveCreateRequests}
+        {...props}
+      />
+    </Harness>
+  </MemoryRouter>
+);
+
+const renderPackageCreateRoute = (props = {}) => render(getPackageCreateRoute(props));
+
+describe('Given PackageCreateRoute', () => {
+  beforeEach(() => {
+    mockGetAccessTypes.mockClear();
+    mockCreatePackage.mockClear();
+    mockRemoveCreateRequests.mockClear();
+    mockHistory.replace.mockClear();
+  });
+
+  afterEach(cleanup);
+
+  it('should handle getAccessTypes', async () => {
+    await act(async () => {
+      await renderPackageCreateRoute({
+        props:{ getAccessTypes: mockGetAccessTypes },
```

Comment:
Please, don't forget spaces after keys in objects:
```suggestion
        props: { getAccessTypes: mockGetAccessTypes },
```